Gemfile

```diff
@@ -31,6 +31,7 @@ gem 'elasticsearch-model'
 gem 'fog-aws', '~> 2.0.0'
 gem 'formatted-dates'
 gem 'has_scope', '~> 0.7.1'
+gem 'image_processing'
```

Comment:
We need to make sure if imagemagick is installed.

Reply:
we'll need an .ebextention to cover that